iree/compiler/Translation/LLVM/Transforms/test/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+iree_lit_test_suite(
+  NAME
+    lit
+  SRCS
+    "expr.mlir"
+  DATA
+    iree::tools::IreeFileCheck
+    iree::tools::iree-opt
+    iree::tools::iree-translate
```

Comment:
Here is the corresponding line where I thought it might be `mlir-translate`, since the Bazel BUILD file has a dep on it and not on `iree-translate.`

Reply:
Taking a look into `expr.mlir`, it seems that neither a dep on `iree-translate` (in the CMake file) nor on `mlir-translate` (in the Bazel file) is necessary so far, since the test only runs `iree-opt` and `IreeFileCheck`.